_posts/2017-02-13-how-to-run-multiple-dependency-builds-on-your-ci-server.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "How to run multiple dependency builds on your CI server"
+date: 2017-02-13 15:06:00
+categories: ["continuous-integration"]
+author: "luciano"
+---
+
+We all know the importance of having in your projects a [CI](https://en.wikipedia.org/wiki/Continuous_integration) tool that allows you to run your test suite. A project shouldn't be running in production without something like that.
+If you are in the situation that you have a core project and many projects that depend on it, and you want to run the tests of those in a single build, I recommend you to keep reading.
+
+<!--more-->
+
+In this particular article, we will be explaining how to do that with [CircleCI](https://circleci.com) and two Ruby projects (a common application and an engine).
```

Comment:
@lubc I'd say: "I will explain how to do that with [CircleCI](https://circleci.com) and two Rails projects (an application and an [engine](http://guides.rubyonrails.org/engines.html))"